src/main/java/org/apache/mesos/hdfs/config/SchedulerConf.java

```diff
@@ -128,16 +128,15 @@ public double getTaskCpus(String taskName) {
   }
 
   public int getJournalNodeCount() {
-    return getConf().getInt("mesos.hdfs.journalnode.count", 1);
+    return getConf().getInt("mesos.hdfs.journalnode.count", 3);
   }
 
-  public String getClusterName() {
-    return getConf().get("mesos.hdfs.cluster.name", "mesos-ha");
+  public String getFrameworkName() {
+    return getConf().get("mesos.hdfs.framework.name", "hdfs");
   }
 
-  // TODO(elingg) This needs to be increased.
   public long getFailoverTimeout() {
-    return getConf().getLong("mesos.failover.timeout.sec", 0);
+    return getConf().getLong("mesos.failover.timeout.sec", 604800);
```

Comment:
I think HDFS should be different than Marathon/Chronos. Let's consider this. :smile: